_episodes/09-mindset.md

```diff
@@ -102,94 +107,102 @@ highlighting our ability to improve with effort.
 {: .challenge}
 
 
-## Errors are Essential to Learning 
+## Errors are Essential to Learning
 
 > The typos are the pedagogy.  
 > — Emily Jane McTavish
 {: .quotation}
 
-One of the barriers to learning is avoidance of making errors. Errors are associated with negative emotions, which leads to learners being 
-fearful of making them. *Error framing* is the process of presenting errors as an integral part of the learning process and using them as teaching opportunities. Error framing encourages learners to understand that making errors provides valuable learning opportunities instead 
+Another example of how our beliefs (in this case, our emotions) impact learning
+is in how we think about errors.  One of the barriers to learning is avoidance of making errors. Errors are associated with negative emotions, which leads to learners being
+fearful of making them.
+
+*Error framing* is the process of presenting errors as an integral part of the learning process and using them as teaching opportunities. Error framing encourages learners to understand that making errors provides valuable learning opportunities instead
```

Comment:
Not a comment on your change, but I think we need a "positive" in here -- "Positive error framing" is what is described, as errors can also be (and often are by default) framed negatively.